backend/server/manual_fixes.py

```diff
@@ -1,22 +1,22 @@
+# pylint: disable=missing-module-docstring,missing-function-docstring
 from contextlib import suppress
+from server.routers.model import StructureContainer
 
 
-def remove_course(structure: dict, course: str) -> dict[str, dict] | None:
+def remove_course(structure: dict[str, StructureContainer], course: str):
     """indescriminantly remove a course from structure"""
```

Comment:
I was going to comment on the spelling here but in doing so, I failed to spell indiscriminately wrong more times than I want to admit. To be honest,the "dictionary spelling" of the word is wrong, I don't know who came up with it but, I wish them a life of pain

Reply:
yea, i will commit this mistake to history now. Whoever has no sin, cast the first stone 😆